lib/PDF/API2/Resource/XObject/Image/PNM.pm

```diff
@@ -12,6 +12,8 @@ use IO::File;
 use PDF::API2::Util;
 use PDF::API2::Basic::PDF::Utils;
 
+use strict;
+no strict 'subs';
```

Comment:
Do you remember why you added this line?  Removing it doesn't seem to cause any compile-time errors on 5.8.5, 5.24.0, or 5.22.1.